### ISSUE TITLE
v2.6.1 is not finding laravel application bootstrap

### ISSUE THREAD

**Issue by ibrunotome** [swooletw/laravel-swoole#192]
**1. Please provide your PHP and Swoole version. (`php -v` and `php --ri swoole`)**

```bash
PHP 7.2.13 (cli) (built: Dec 10 2018 18:43:50) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.13, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.6.1, Copyright (c) 2002-2018, by Derick Rethans
```

```bash
Swoole => enabled
Author => Swoole Team <team@swoole.com>
Version => 4.2.12
Built => Jan 10 2019 17:31:49
coroutine => enabled
epoll => enabled
eventfd => enabled
signalfd => enabled
cpu_affinity => enabled
spinlock => enabled
rwlock => enabled
mutex_timedlock => enabled
pthread_barrier => enabled
futex => enabled
async_redis => enabled

Directive => Local Value => Master Value
swoole.enable_coroutine => On => On
swoole.display_errors => On => On
swoole.use_shortname => On => On
swoole.fast_serialize => Off => Off
swoole.unixsock_buffer_size => 8388608 => 8388608
```
**2. Please provide your Laravel/Lumen version.**

Laravel 5.7.20

**3. Which release version of this package are you using?**

v2.6.1

**4. What did you do? If possible, provide a recipe for reproducing the error.**

Just updated to v2.6.1 and after that, up my docker normally and see the message:

<img width="1356" alt="screenshot 2019-01-13 19 33 51" src="https://user-images.githubusercontent.com/4256471/51090895-31252180-176a-11e9-9da2-9d43960f6d8e.png">

**5. What did you expect to see?**

Before that, I was using swooletw v2.5.0 and everything worked normally

**Comment by ibrunotome** [comment]
Just to be sure, I did a rollback for v2.5.0 and works fine.

**Comment by albertcht** [comment]
Hi @ibrunotome ,

Thanks for your report. This has been fixed in `v2.6.2`
